medusa/themes/base.py

```diff
@@ -0,0 +1,73 @@
+import logging
+import json
+import os
+from medusa import app
+from medusa.logger.adapters.style import BraceAdapter
+
+
+log = BraceAdapter(logging.getLogger(__name__))
+log.logger.addHandler(logging.NullHandler())
+
+themes = []
+
+
+class Theme(object):
+    """Base theme class."""
+
+    def __init__(self, name, version, dep_version=None, author=None):
+        self.name = name
+        self.version = version
+        self.dep_version = dep_version
+
+
+def read_themes():
+    """Read the theme folder and create a Theme object for every entry, if valid."""
```

Comment:
I'm using this new module to read and check the themes.
If a theme has a configuration error, it will report the error and exit medusa.